pixi.toml

```diff
@@ -0,0 +1,28 @@
+[project]
+name = "iris-hep.github.io"
+version = "0.1.0"
+description = "Jekyll based website for IRIS-HEP"
+authors = ["Matthew Feickert <matthew.feickert@cern.ch>"]
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-64", "osx-arm64", "win-64"]
```

Comment:
It would be cool if we could find someone with a Windows machine to test this. I could do it in CI, but that requires additional changes.